pkg/controllers/clusterschedulingpolicysnapshot/controller.go

```diff
@@ -33,48 +32,58 @@ import (
 	"github.com/kubefleet-dev/kubefleet/pkg/utils/controller"
 )
 
-// Reconciler reconciles a clusterSchedulingPolicySnapshot object.
+// Reconciler reconciles a clusterSchedulingPolicySnapshot or schedulingPolicySnapshot object.
 type Reconciler struct {
 	client.Client
 
 	// PlacementController exposes the placement queue for the reconciler to push to.
 	PlacementController controller.Controller
 }
 
-// Reconcile triggers a single CRP reconcile round when scheduling policy has changed.
+// Reconcile triggers a single placement reconcile round when scheduling policy has changed.
 func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	name := req.NamespacedName
-	snapshot := fleetv1beta1.ClusterSchedulingPolicySnapshot{}
-	snapshotKRef := klog.KRef(name.Namespace, name.Name)
+	var snapshot fleetv1beta1.PolicySnapshotObj
```

Comment:
move to line 55, closer to where they're used

Reply:
fixing with next PR.